### DISCUSSION TITLE
How does cpd specify scanning in multiple languages

### DISCUSSION THREAD

**Discussion by 2714222609** [pmd/pmd#4451] [Q&A]
How does cpd specify scanning in multiple languages? I want to scan c,cpp,python,java in my project, the command below doesn't work.

pmd cpd --language c,cpp,python,java
pmd cpd --language "c,cpp,python,java"

What's the correct command?

**Comment by jsotuyod** [comment]
CPD doesn't currently support this. The main PMD analysis does, but not CPD yet. You would need one run per language.

**Reply by oowekyala** [reply]
(just noting that #4397 makes this possible by not relying on a single static token repository. It would be possible then to create one tokenizer per language)
